src/pages/Delivery/Outside/DetailAddress.tsx

```diff
@@ -0,0 +1,177 @@
+  const getCustomInput = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setCustomInputValue(e.target.value.trimStart());
+  }, []);
+
+  const handleSubmitRequest = () => handleCloseBottomModal();
+
+  return (
+    <div className="flex flex-col items-center">
+      <div className="p-1"> 위에 구분</div>
+      <div className="shadow-subtle w-[21.375rem] rounded-xl">
+        <div id="map" className="h-40 w-full rounded-t-xl border border-neutral-300"></div>
+        <div className="flex h-[3.5rem] w-full items-center justify-between rounded-b-xl bg-white px-6 text-[0.813rem] text-neutral-600">
+          충남 천안시 동남구 병천면 가전8길 102
+          <div>
+            <ArrowGo />
+          </div>
+        </div>
+      </div>
+      <div className="mt-[1.813rem]">
+        <div className="text-primary-500 leading-[160%] font-semibold">상세 주소</div>
+        <div className="pb-3 text-xs leading-[160%]">상세 주소를 입력해주세요</div>
+        <input
+          className="w-[21.375rem] rounded-sm border border-neutral-300 bg-white px-4 py-3 placeholder-neutral-400 placeholder:text-sm"
+          type="text"
+          name="DetailAddress"
+          placeholder="상세주소를 입력해주세요 (건물명, 동/호수 등)"
+        />
+      </div>
+      <div className="mt-[1.813rem]">
+        <div className="text-primary-500 pb-2 font-semibold">배달기사님에게</div>
+        <button
+          type="button"
+          onClick={handleOpenBottomModal}
+          className="flex w-[21.375rem] items-center justify-between rounded-sm border border-neutral-300 bg-white px-4 py-3 text-sm text-neutral-400"
+        >
+          {requestLabel()}
+          <div className="pointer-events-none">
+            <ArrowDown />
+          </div>
+        </button>
+      </div>
+      <button
+        onClick={handleOpenModal}
+        className="bg-primary-500 fixed bottom-4 h-[2.875rem] w-[21.375rem] rounded-lg font-semibold text-white"
+      >
+        주소 선택
+      </button>
+      <Modal isOpen={modalIsOpen} onClose={handleCloseModal}>
+        <ModalContent>정확한 상세 주소를 입력해주세요.</ModalContent>
+        <ModalFooter>
+          <button
+            onClick={handleCloseModal}
+            className="bg-primary-500 h-12 w-[16rem] rounded-lg font-semibold text-white"
+          >
+            확인
+          </button>
+        </ModalFooter>
+      </Modal>
+      <BottomModal isOpen={bottomModalIsOpen} onClose={handleCloseBottomModal}>
+        <BottomModalHeader>
+          <div className="text-primary-500 font-semibold">배달기사님에게</div>
+          <button onClick={handleCloseBottomModal}>
+            <CloseIcon />
+          </button>
+        </BottomModalHeader>
+        <BottomModalContent>
+          <form className="flex w-[20.375rem] flex-col gap-2">
+            {DetailRequest.map((detail, index) => (
+              <label
+                key={index}
+                htmlFor={detail}
+                className={`request-label ${selectRequest === detail ? 'request-label-checked' : ''}`}
+              >
```

Comment:
이항 연산자로 구분되는 로직은 아닌 것 같은데 아래와 같이는 안될까요?

```suggestion
                className={`request-label ${selectRequest === detail && 'request-label-checked'}`}
              >
```

Reply:
수정하겠습니다 감사합니다